### ISSUE TITLE
Ajax Sort

### ISSUE THREAD

**Issue by airtonix** [grid-js/gridjs#50]
**Is your feature request related to a problem? Please describe.**
Sorting needs to sort the entire dataset, not just what's in view.

**Describe the solution you'd like**
Ideally you should make the api available to the custom functions: 

```js
async data () { 
  const sorting = this.getSortingParameters()
  await axios.get('things', { params: { sorting } )
},
async sort () {
  // do sorting things
 return { ...someKindOfSortingParameters }
}
```
..
**Additional context**
This will be needed if you want grid.js to have any serious traction in admin dashboards (create update delete)

**Comment by afshinm** [comment]
Makes sense. I'm currently working on connecting other components to the Server storage and will release a version soon. thanks!
